### PR TITLE
doc: prepare new release 4.1.3

### PR DESCRIPTION
**Pull request by knodelle**
#5477

## Checkpoints

**For developers**

- [x] If relevant, manual tests have been done to ensure the stability of the whole application and that the involved feature works
- [x] The original issue is up to date w.r.t the latest discussions and contains a short summary of the implemented solution
- [x] Automated tests covering the feature or the fix, have been written, deemed irrelevant (give the reason), or an issue has been created to implement the test (give the link)
- [x] Make sure that:
  - if there are modifications in the Tracim configuration files (eg. `development.ini`), they are documented in `backend/doc/setting.md`
  - any migration process required for existing instances is documented
  - relevant people for these changes are notified
- [x] Original authors of the features included in a multi-feature branch (maintenance fixes -> develop, security fixes -> develop, …) should be part of the reviewers, especially if you encountered merge conflicts.

**For code reviewers**

- [x] The code is clear enough
- [x] If there are FIXMEs in the code, related issues are mentioned in the FIXME
- [x] If there are TODOs, NOTEs or HACKs in code, the date and the developer initials are present

**For testers**

- Manual, quality tests have been done

### CODE REVIEW

File: changelog.md [hunk 1]
```diff
@@ -1,3 +1,13 @@
+# 4.1.3 / 2022-03-01
+
+### Fixed Issues
+
+- Frontend: [#5430](https://github.com/tracim/tracim/issues/5430),
+[#5453](https://github.com/tracim/tracim/issues/5453),
+[#5455](https://github.com/tracim/tracim/issues/5455)
+
+
+
```
Comment (PhilippeAccorsi) [inline changelog.md:10]: Remove one line (juste two line necessary here).

**Review by gbonaspetti — APPROVED** [review]

**Review by PhilippeAccorsi — CHANGES_REQUESTED** [review]

**Comment by PhilippeAccorsi** [comment]
@knodelle issue not close in project board.

File: changelog.md (hunk 1 above)
Comment (knodelle) [reply changelog.md]: done, thank  you

**Comment by knodelle** [comment]
> @knodelle issue not close in project board.

I usually close them after tagging, but I can close them now @PhilippeAccorsi

**Review by inkhey — APPROVED** [review]
Changelog look ok to me:
- don't forget to update the board (all should be in done)
- is there any reason to have sometime description for maintenance release and sometime not ? ( related to how important are the fix ?)

**Comment by PhilippeAccorsi** [comment]
> Changelog look ok to me:
> 
>     * don't forget to update the board (all should be in done)
> 
>     * is there any reason to have sometime description for maintenance release and sometime not ? ( related to how important are the fix ?)

The actually rules is to not add description for maintenance (is it done by "mistake" from one dev).

**Comment by PhilippeAccorsi** [comment]
> > @knodelle issue not close in project board.
> 
> I usually close them after tagging, but I can close them now @PhilippeAccorsi

I think it is better to close issue when sort of all issue is done (to make sure when this pr is merged, just related issue need to be closed.)

**Review by PhilippeAccorsi — APPROVED** [review]

**Comment by PhilippeAccorsi** [comment]
@knodelle if you merge this PR tomorrow, please change date of release in the changelog.md file.

**Comment by PhilippeAccorsi** [comment]
We need to use the same date in the changelog and for the tag.

**Pull request merged**